contrib/libdbusmenu-glib/template.py

```diff
@@ -0,0 +1,75 @@
+pkgname = "libdbusmenu-glib"
+pkgver = "16.04.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--disable-dumper",
+    "--disable-gtk-doc",
+    "--enable-tests",
+    "--enable-static",
+    "--enable-gtk",
+    "--enable-introspection",
+    "--enable-vala",
+    "HAVE_VALGRIND_FALSE=''",
+]
+make_cmd = "gmake"
+hostmakedepends = [
+    "autoconf",
+    "automake",
+    "gmake",
+    "intltool",
+    "libtool",
+    "pkgconf",
+]
+makedepends = [
+    "atkmm-devel",
+    "gettext-devel",
+    "gtk-doc-tools",
+    "gtk+3-devel",
+    "glib-devel",
+    "gobject-introspection",
+    "json-glib-devel",
+    "vala-devel",
+]
+pkgdesc = "Library for passing menus over DBus"
+maintainer = "avgwst <avgwst@tutanota.de>"
+license = "GPL-3.0-or-later AND LGPL-3.0-or-later AND LGPL-2.1-or-later"
+url = "https://launchpad.net/libdbusmenu"
+source = f"{url}/16.04/{pkgver}/+download/libdbusmenu-{pkgver}.tar.gz"
+sha256 = "b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a"
+# vis breaks symbols
```

Comment:
drop this, vis is only useful for cfi and we're never enabling that for gtk/glib things